textile/chat-features.textile

```diff
@@ -404,10 +432,13 @@ Presence allows chat room users to indicate to others that they're online, as we
 ** @(CHA-PR7a)@ @[Testable]@ Users may provide a listener to subscribe to all "presence events":#chat-structs-presence-event in a room.
 ** @(CHA-PR7b)@ @[Testable]@ Users may provide a listener and a list of selected "presence events":#chat-structs-presence-event, to subscribe to just those events in a room.
 ** @(CHA-PR7c)@ @[Testable]@ A subscription to presence may be removed, after which it shall receive no further events.
-* @(CHA-PR8)@ @[Testable]@ Users may subscribe to discontinuity events to know when there's been a break in presence. Their listener will be called when a discontinuity event is triggered from the room lifecycle. For presence, there shouldn't need to be user action as the underlying core SDK will heal the presence set.
+* @(CHA-PR8)@ This specification point has been removed. It was valid up until the single-channel migration.
 * @(CHA-PR9)@ Users may configure their presence options via the @RoomOptions@ provided at room configuration time.
-** @(CHA-PR9a)@ @[Testable]@ If @enter@ is set to true, then the presence feature must set the @PRESENCE@ mode on the underlying Realtime channel. The default value is true.
-** @(CHA-PR9b)@ @[Testable]@ If @subscribe@ is set to true, then presence feature must set the @PRESENCE_SUBSCRIBE@ mode on the underlying realtime channel. The default value is true. Note that setting this to false does not prevent the client from receiving its own presence messages, but it will not receive them from others.
+** @(CHA-PR9a)@ This specification point has been removed. It was valid up until the single-channel migration and was replaced by @CHA-PR9c@.
+** @(CHA-PR9b)@ This specification point has been removed. It was valid up until the single-channel migration and was replaced by @CHA-PR9c@.
+** @(CHA-PR9c)@ The option @receivePresenceEvents@ controls whether or not the client should receive presence events.
+*** @(CHA-PR9c1)@ @[Testable]@ This option defaults to @true@.
+*** @(CHA-PR9c2)@ @[Testable]@ When set to @false@, when getting the channel from the realtime client, the @PRESENCE_SUBSCRIBE@ channel mode shall be omitted.
 
 h2(#typing). Typing Indicators
```

Comment:
Should we not update (or ideally remove per https://github.com/ably/specification/pull/282/files#r2012264825) CHA-T1?

Reply:
Yes, I've done this in 151c0904